en/getting_started/quick_start.md

```diff
@@ -1,47 +1,21 @@
-# This is QGroundControl
+# QGroundControl Quick Start
 
-The first step is to [download](../getting_started/download_and_install.md) and install QGroundControl. Below is the screen you will see when no vehicle is connected.
+Getting *QGroundControl* up and running is quick and easy:
 
-![](../../images/quickstart/BootToFly.jpg)
-<br>
+1. [Download and install](../getting_started/download_and_install.md) the application.
+1. Start *QGroundControl*. 
+1. Attach your vehicle to the ground station device via USB, through a telemetry radio, or over WiFi. *QGroundControl* should detect your vehicle and connect to it automatically.
 
-![](../../images/quickstart/SettingsViewButton.jpg) **[Settings](../SettingsView/SettingsView.md)**
-<br>Configure the QGroundControl application.
+That's it! If the vehicle is ready to fly,  *QGroundControl* should display [Fly View](../FlyView/FlyView.md) as shown below (otherwise it will open [Setup View](../SetupView/SetupView.md)).
 
-![](../../images/quickstart/SetupViewButton.jpg) **[Setup](../SetupView/SetupView.md)**
-<br>Configure and tune your vehicle.
+![](../../images/quickstart/fly_view_connected_vehicle.jpg)
 
-![](../../images/quickstart/PlanViewButton.jpg) **[Plan](../PlanView/PlanView.md)**
-<br>Create autonomous missions.
+A good way to become familiar with *QGroundControl* is to start experimenting:
```

Comment:
I don't know about GitBook but in MacDown app the fact that there is no blank link after this screws up the formatting of the bullet list.